### PR TITLE
Use encode / decode methods in twig template

### PR DESCRIPTION
**Pull request by stephledev**
Since I use hashids quite often to generate paths in twig templates, I thought it would be a good idea to add a twig extension.

### CODE REVIEW

**Comment by roukmoute** [comment]
Hey !

Of course this is a good idea :smile:
It was in todo list but I do not take the time.

Thanks!

File: src/Twig/HashidsExtension.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    /**
+     * @var \Hashids\Hashids;
+     */
+    private $hashids;
+
+    public function __construct(Hashids $hashids)
+    {
+        $this->hashids = $hashids;
+    }
+    public function getFilters()
```
Comment (roukmoute) [inline src/Twig/HashidsExtension.php]: missing return.

File: src/Twig/HashidsExtension.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Roukmoute\HashidsBundle\Twig;
+
+use Roukmoute\HashidsBundle\Hashids;
+
+class HashidsExtension extends \Twig_Extension
+{
+    /**
+     * @var \Hashids\Hashids;
```
Comment (roukmoute) [inline src/Twig/HashidsExtension.php]: Is this not simply `Hashids`?

Comment (stephledev) [inline src/Twig/HashidsExtension.php]: yes, you're right since `Roukmoute\HashidsBundle\Hashids` already expands `\Hashids\Hashids`

**Comment by roukmoute** [comment]
Thanks !

**Pull request merged**